### ISSUE TITLE
Discrepancy between Akahu User model and API response

### ISSUE THREAD

**Issue by nevercast** [akahu-io/akahu-sdk-js#48]
Hello,

I've noticed a discrepancy between the Akahu User model and the actual response from the API.

According to the Akahu User model, the user object should have the following properties:

```typescript
export declare type User = {
    _id: string;
    access_expires_at: string;
    first_name?: string;
    last_name?: string;
    preferred_name?: string;
    email?: string;
    mobile?: string;
};
```

However, when I query the endpoint, I get the following response:

```json
{
  "_id": "redacted",
  "access_granted_at": "2023-10-21T20:35:21.033Z",
  "preferred_name": "redacted",
  "email": "redacted"
}
```

As you can see, the actual response from the API includes an `access_granted_at` property, not `access_expires_at` as defined in the Akahu User model.

Could you please clarify this discrepancy? Is the `access_expires_at` field deprecated, or is this a mistake in the model?

Thank you for your attention to this matter.

**Comment by dabrowne** [comment]
Hi @nevercast thanks for reporting this discrepancy. It looks like there have been a couple of changes since the schema was originally modeled that haven't been captured here.

- `access_expires_at` no longer exists (our enduring access consent is indefinite)
- `access_granted_at` is new and has now been added to the type definition

We've just released a new version `akahu@1.15.2` that includes these changes.
